support/cas-server-support-couchdb-core/src/main/java/org/apereo/cas/couchdb/core/CouchDbConnectorFactory.java

```diff
@@ -50,17 +103,24 @@ public CouchDbConnector create() {
             .cleanupIdleConnections(couchDbProperties.isCleanupIdleConnections());
 
 
-        if (!StringUtils.isBlank(couchDbProperties.getUsername())) {
-            builder.username(couchDbProperties.getUsername());
+        if (StringUtils.isNotBlank(couchDbProperties.getProxyHost())) {
+            builder.proxy(couchDbProperties.getProxyHost());
+
+            if (couchDbProperties.getProxyPort() > 0) {
+                builder.proxyPort(couchDbProperties.getProxyPort());
+                LOGGER.warn("CouchDb proxy settings enabled [{}:{}]", couchDbProperties.getProxyHost(), couchDbProperties.getProxyPort());
+            } else {
+                LOGGER.error("Proxy port not set for host [{}] clearing proxy host.", couchDbProperties.getProxyHost());
+                builder.proxy(null);
```

Comment:
Are the above log statements really, errors and warnings? Seems like debug and info should suffice just fine.

Reply:
Dont